ubuntudesign/www.ubuntu.com
# Change $asset-server to $asset-server-url

Change the name of the SASS setting variable to match the naming used in templates.

QA
--

Run the site, check all CSS looks fine, especially links to assets.ubuntu.com in CSS.
